### PR TITLE
Order history

### PR DESCRIPTION
**Pull request by ShahirAliGit**
Pull request for associated Jira issue

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -86,6 +86,39 @@ def activityhistory
...
+
+    end
+
+    spend.each do |s|
+      reward = Reward.find_by(id: s.reward_id)
+      user = User.find_by(id: s.user_id)
+
+      if reward && user
+        #             Tyoe  ,user id,   user name,  reward id, reward name,         points,     created at,   updated at
+        @history << ["Spent", user.id, user.name, reward.id, reward.name, -1 * reward.point_value, s.created_at, s.updated_at]
```
Comment (CerebralDatabank) [inline app/controllers/users_controller.rb:113]: As we discussed:
In this line, multiplying the point value by −1 creates an invalid value. The minus should just be tacked onto the left when creating the string.

**Review by CerebralDatabank — CHANGES_REQUESTED** [review]
Got this error when clicking History:
image
As we discussed, it's a quick fix regarding the −1 multiplication in the code.

**Review by CerebralDatabank — APPROVED** [review]
LGTM! 👍

**Review by annapachecohartman — APPROVED** [review]
LGTM!

**Review by jhbeeber — APPROVED** [review]
Looks good to me. Just add fixes for the rubocop errors and it should be good to merge.

**Pull request merged**